### PR TITLE
trivial:

### PR DESCRIPTION
**Pull request by truongsinh**
- fix google sheet link
- add build badge

### CODE REVIEW

**Comment by duynht** [comment]
Seems good! Let's merge!

File: README.md [hunk 1]
```diff
@@ -237,3 +238,5 @@ Also check out the site-shared
 [Repo on Travis]: https://travis-ci.org/flutter/website
 [rvm]: https://rvm.io/rvm/install#installation
 [this repo]: https://github.com/flutter/website
+[Netlify Status]: https://api.netlify.com/api/v1/badges/c2329679-dfdf-4467-8a41-d5430cead699/deploy-status
```
Comment (duynht) [inline README.md:241]: Demo code review?

**Review by duynht — APPROVED** [review]
Seems good! Let's merge!

File: README.md (hunk 1 above)
Comment (truongsinh) [reply README.md:241]: yes

**Pull request merged**